### PR TITLE
histogram: update documentation

### PR DESCRIPTION
**Pull request by dtorop**
- New images
- Describe RGB Parade
- Talk about histogram profile
- More information on waveform
- Describe how to resize/hide the panel

### CODE REVIEW

File: doc/usermanual/darkroom/panels/histogram.xml [hunk 1]
```diff
@@ -16,36 +16,244 @@
...
+		is <emphasis>Shift+Ctrl+H</emphasis>.
+	      </para>
+
+	      <para>
+		It is possible to assign keyboard shortcuts to cycle through or switch histogram
+		mode/type. Separate shortcuts can be set for the darkroom and tethering
+		views. By default, none of these shortcuts are associated with keys.
+	      </para>
+
+	      <para>
```
Comment (TurboGit) [inline doc/usermanual/darkroom/panels/histogram.xml:64]: This is really for the releases notes and should not be part of the documentation I would say.

Comment (dtorop) [reply doc/usermanual/darkroom/panels/histogram.xml:64]: @TurboGit Were you thinking to remove the entire paragraph ("It is possible to assign...none of these shortcuts are associated.") or just the last sentence ("By default...with keys.")? I'm happy to remove either/both...

**Comment by dtorop** [comment]
@TurboGit: I'll re-revise as needed, but does this address your thoughts on this?

File: doc/usermanual/darkroom/panels/histogram.xml (hunk 1 above)
Comment (TurboGit) [reply doc/usermanual/darkroom/panels/histogram.xml]: Sorry a bit late, my idea was to remove only:

`The histogram is currently disabled in tethering view, pending a code fix.`

The other paragraph are ok to me.

Comment (dtorop) [reply doc/usermanual/darkroom/panels/histogram.xml:64]: OK! I did this, I hope...

**Comment by dtorop** [comment]
Great! Hope that histogram work is tidy for v3.2. Should be more to come for v.3.4...

**Pull request merged**